pegasus/sites.v3/code.org/public/educate/weblab-test.html

```diff
@@ -0,0 +1,220 @@
+          department to update your school's firewall settings or device
+          policies.
+        </p>
+        <p>
+          When checking network support, try to meet as many of these criteria
+          as possible:
+        </p>
+        <li>Use your school's internet connection</li>
+        <li>Use a student computer</li>
+        <li>Log into the computer with a student account</li>
+        <br />
+        <p>
+          Note: You do not need to be logged in to Code.org for this test to be
+          successful.
+        </p>
+        <label for="connect"
+          >Click "Connect" to check whether your school's network supports the
+          Web Lab tool</label
+        >
+        <div id="status-table">
+          <table>
+            <tbody>
+              <tr>
+                <th>Check</th>
+                <th>Status</th>
+              </tr>
+              <tr>
+                <td>
+                  Connected to
+                  <code>https://downloads.computinginthecore.org</code>
+                </td>
+                <td class="computinginthecore">
+                  <p class="incomplete">Not complete</p>
+                  <p class="loading" style="display: none">Connecting...</p>
+                  <p class="succeeded" style="display: none">Success</p>
+                  <p class="failed" style="display: none">Failed</p>
+                </td>
+              </tr>
+              <tr>
+                <td>Connected to <code>https://codeprojects.org</code></td>
+                <td class="codeprojects">
+                  <p class="incomplete">Not complete</p>
+                  <p class="loading" style="display: none">Connecting...</p>
+                  <p class="succeeded" style="display: none">Success</p>
+                  <p class="failed" style="display: none">Failed</p>
+                </td>
+              </tr>
+              <tr>
+                <td>Able to load an iFrame</td>
+                <td class="iframe">
+                  <p class="incomplete">Not complete</p>
+                  <p class="loading" style="display: none">Connecting...</p>
+                  <p class="succeeded" style="display: none">Success</p>
+                  <p class="failed" style="display: none">Failed</p>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <div>
+          <button id="connect" class="btn btn-primary">Connect</button>
+        </div>
+        <p id="success-message" style="display: none">
+          <strong>You passed all checks! No further action is needed.</strong>
+        </p>
+        <div id="fail-message" style="display: none">
+          <p>
+            <strong>Error. One or more checks failed.</strong>
+          </p>
+          <p>
+            <strong>Action required: </strong>
+            Please contact your IT department to update your school’s firewall
+            settings or device policies. Here is the rest of this call to
```

Comment:
Should "Here is the rest..." be here?

Reply:
The copy is better without it, thank you!